package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-loader": "^2.1.1",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-railstats": "file:./eslint",
```

Comment:
This line is causing an error as there is no file at this location - did you mean .eslintrc.js ?

Reply:
or maybe forgot to commit that file?

Reply:
Good call. I was experimenting with something but must've forgot to remove everything from it